jobrunner/sync.py

```diff
@@ -0,0 +1,116 @@
+"""
+Script which polls the job-server endpoint for active JobRequests and POSTs
+back any associated Jobs.
+"""
+import time
+
+import requests
+
+from .log_utils import configure_logging, set_log_context
+from . import config
+from .create_or_update_jobs import create_or_update_jobs
+from .database import find_where
+from .models import JobRequest, Job
+from .string_utils import slugify, project_name_from_url
+
+
+session = requests.Session()
+
+
+def main():
+    while True:
+        sync()
+        time.sleep(config.POLL_INTERVAL)
+
+
+def sync():
+    results = api_get(
+        "job-requests", params={"active": "true", "backend": config.BACKEND}
+    )
```

Comment:
Should there be some explicit query param setting for number of jobs to request in one batch? If the answer is "as many as possible, leave it to the server" then is that worth mentioning here?